elastica/rod/cosserat_rod.py

```diff
@@ -520,6 +520,164 @@ def compute_shear_energy(self):
             * (_batch_dot(sigma_diff, shear_internal_forces) * self.rest_lengths).sum()
         )
 
+    def compute_position_of_point(self, point, index):
```

Comment:
Let's remove `compute_position_of_point` and  `compute_velocity_of_point` from Cosserat Rod and include them in joint file as functions.

Reply:
Implemented in https://github.com/GazzolaLab/PyElastica/pull/149/commits/74a9c9d3f1511d9aed73a7f7f7b804cf0bebb284